### PR TITLE
Change favicon + Combine homepage and aboutus

### PR DESCRIPTION
**Pull request by PhuocOng**
- Changing Favicon for the website
- Combining home.jsx and aboutus.jsx into Home.jsx

https://github.com/viettech-group3/productify/assets/122703392/895dadd4-1670-4bb4-9f91-a8d9b05117ef

### CODE REVIEW

File: frontend/src/components/Home/HeroSection.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Signup from "../../components/Authentication/Signup";
 
 const HeroSection = () => {
   return (
-    <div className={styles.container}>
+    <div className={`${styles.container}`}>
       <div className={`${styles.gradientBackground}`}>
         <div>
           <div className={styles.content}>
```
Comment (khoa165) [inline frontend/src/components/Home/HeroSection.jsx:13]: why the use of backtick \`?
Would it still work without it? If so, then you probably don't want to keep it there
Also inconsistent here `<div className={styles.content}>`

File: frontend/src/pages/Home.jsx [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 
 const Home = () => {
   return (
     <div>
-      <Navbar/>
-        <HeroSection>HeroSection</HeroSection>
+      <Navbar />
+      <HeroSection>HeroSection</HeroSection>
+      <Div2/>
+      <Div1/>
```
Comment (khoa165) [inline frontend/src/pages/Home.jsx:14]: I don't like these namings. Components should have description names em. If I look at this code, I have no idea what those divs contain.

**Review by khoa165 — COMMENTED** [review]
Some minor issues but overall good work em

File: frontend/src/components/Home/HeroSection.jsx (hunk 1 above)
Comment (PhuocOng) [reply frontend/src/components/Home/HeroSection.jsx:13]: Oh thank you!
Em edit code đoạn sau mà quên xóa backtick ` ạ, cảm ơn anh nhiều hehe

File: frontend/src/pages/Home.jsx (hunk 2 above)
Comment (PhuocOng) [reply frontend/src/pages/Home.jsx:14]: Thank you for your feedback!
I'm gonna edit it right now

**Comment by lhoang25** [comment]
Lovelyyyy work! We can consider using global variables for color names in global.css so we don't have to look up every single color code in the future.

**Review by lhoang25 — APPROVED** [review]
Lovelyyyy work! We can consider using global variables for color names in global.css so we don't have to look up every single color code in the future.

**Review by khoa165 — APPROVED** [review]
LGTM

**Pull request merged**